templates/10-group-validation.ValidatingWebhookConfiguration.yaml.tmpl

```diff
@@ -0,0 +1,52 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata: 
+  name: sre-validate-webhook
```

Comment:
Shouldn't `metadata.name` refer to group-validation?

Reply:
Yep. It should also have only one webhook 😊 I'll fix both.